### PR TITLE
Hardware Testing scripts documentation - UI

### LINKED ISSUE #536: Commissioning scripts
Update and integrate explanation showing how commissioning scripts work into the current docs.

### PR DESCRIPTION
**Pull request by degville**
This is the first of 3 PRs to fix #536, split to make reviews easier.

### CODE REVIEW

File: en/nodes-hw-scripts.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+The above Bash script contains comment-delineated metadata plus two lines of
+functionality, the first to install the `stress-ng` package (a CPU stress-test
+utility) and the second to execute **stress-ng** with various arguments.
+
+The following metadata can be included:
+
+- `name`: Corresponds to the script's filename
+- `title`: Human-friendly descriptive version of name, used within the web UI
+- `description`: Brief outline of what the script does
+- `tags`:  List of tags associated with the script.
```
Comment (evilnick) [inline en/nodes-hw-scripts.md]: full stops or not full stops...

File: en/nodes-hw-scripts.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+- `script-type`,`hardware_type`: Broad categories for the kind of script and
+  what the script tests
```
Comment (evilnick) [inline en/nodes-hw-scripts.md]: Re these predefined anywhere? Are there guidelines? I'm guessing not...

File: en/nodes-hw-scripts.md (hunk 1 above)
Comment (degville) [reply en/nodes-hw-scripts.md]: thanks!

File: en/nodes-hw-scripts.md (hunk 2 above)
Comment (degville) [reply en/nodes-hw-scripts.md]: no, unfortunately. I've added a TODO as there are a few things like this in these docs.

**Pull request merged**